core/block/source/source.go

```diff
@@ -249,6 +252,11 @@ func (s *source) PushChange(params PushChangeParams) (id string, err error) {
 	if err != nil {
 		return
 	}
```

Comment:
can we test it?

Reply:
Yeah, imported any-sync mocks, so could be tested easily!

Reply:
I don't think importing mocks is good idea. You can use mockery to easily generate these mocks and thereby reducing dependencies. Even better you can restructure code so it have transparent business logic without dependencies at all